hoot-core/src/main/cpp/hoot/core/visitors/geometrymodifiers/PolyToWayGeoModifierAction.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of Hootenanny.
+ *
+ * Hootenanny is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * --------------------------------------------------------------------
+ *
+ * The following copyright notices are generated automatically. If you
+ * have a new notice to add, please use the format:
+ * " * @copyright Copyright ..."
+ * This will properly maintain the copyright information. Maxar
+ * copyrights will be updated automatically.
+ *
+ * @copyright Copyright (C) 2019, 2020, 2021 Maxar (http://www.maxar.com/)
+ */
+#ifndef POLYTOWAYGEOMODIFIERACTION_H
+#define POLYTOWAYGEOMODIFIERACTION_H
+
+#include "GeometryModifierAction.h"
+
+namespace hoot
+{
+
+/**
+ * Converts Polygons to Ways.
+ */
+class PolyToWayGeoModifierAction : public GeometryModifierAction
```

Comment:
Where is your code that is going to use this class? I think that might make it easier for me to understand the need for it.